agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml

```diff
@@ -25,6 +25,9 @@ spec:
         hostPath:
           path: /etc/os-release
           type: FileOrCreate
+      - name: etc-version
+        hostPath:
+          path: /etc/VERSION
```

Comment:
I used `type: FileOrCreate` for /etc/os-release above, but if you think /etc/VERSION might also be a directory then maybe that wouldn't be appropriate here

Reply:
/etc/VERSION on the host will either be a file (minikube) or not exist (!minikube). If you mount a nonexistent file anyway, you'll get an empty directory, which is minimally easier for me to detect than a zero-sized file.

I'm happy to do it in any way that makes sense.